hacking/aws_config/build_iam_policy_framework.py

```diff
@@ -183,7 +183,7 @@ def get_tables(service):
     for df_list in html_list:
         for df in df_list:
             table = json.loads(df.to_json(orient='split'))
-            table_data = table['data'][0]
+            table_data = df
```

Comment:
It appears this is a Python 2 versus Python 3 bug (I tested with 2.7 and 3.7). This fixes Python 3 and breaks 2, so it would be nice if this could be fixed in a compatible way instead.

Reply:
Ok. I'm open to suggestions. Just wanted to fix it for my purposes (in Python 3)

Reply:
Still works with Python 2.7 for me. Probably something with your environment